Dockerfile

```diff
@@ -9,9 +9,12 @@ COPY --from=build /app/build ${HTML_DIR}
 COPY nginx/nginx.conf /etc/nginx/conf.d/default.conf
 WORKDIR ${HTML_DIR}
 COPY docker-entrypoint.sh .
+# This will be generated in the pipeline.
+# Comment out if building locally
```

Comment:
Hm, I don't like modifying files that are shared with CI for local development, as there is a risk of committing

Shall I create a ticket?


<details>
  <summary>TASK: Don't attach bom locally</summary>
  
  # Task
  To make dockerfile more local-dev friendly, it should automatically skip adding sbom locally
  
  # Engineering notes
  
Add sbom if `LOCAL_MACHINE=true`

And add `LOCAL_MACHINE=true` to [./scripts/docker-build](https://github.com/telicent-oss/telicent-instance/blob/main/scripts/docker-build#L17)

note: [LOCAL_MACHINE](https://github.com/Telicent-io/telicent-search/pull/493/files#diff-14fa99bf0fc0a5138e2000de66137c572d2c3083a02085d47224f82282e6fb39R17) is a [new standardization](https://xkcd.com/927/) I pulled out of thin air and didn't entirely derisk by checking if there was something appropriate already


</details>